src/crypto/kdf/AesKdf.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ *  Copyright (C) 2017 angelsl
```

Comment:
It would be nice if we wouldn't put individual people in license headers. After a few months or years, many other people will have edited or even completely refactored the file. Rather put KeePassXC Team in here and add yourself to a contributors file (see my comment about this at the beginning of this review).